index.html

```diff
@@ -1,53 +1,182 @@
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-<link rel="stylesheet" href="style.css">
-    <title>technicalsupport</title>
-  </head>
-  <body>
-    <div class="header">
-      <div class="header-content">
-      <span class="pagetitle"></span><div class="logo"><div class="menubar">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css">
+  <title>technicalsupport</title>
+</head>
+
+<body>
+  <section class="header">
+    <div class="header__content">
+      <div class="page__title"></div>
+      <div class="logo"></div>
```

Comment:
Si astuia i-as fi zis `header__logo` si la fel in loc de `page__title` as fi numit `header__title`. De ce? Pentru ca poti sa ai si `footer__logo` sau vreo sectiune cu titlu. E bine sa fii specific cand numesti chestii, caci te cor ajuta ulterior sa identifici repede unde gasesti acele element in pagina. Deci incearca mereu sa numesti ceva gen `[sectiune]__[sectione in sectiune daca e mare]__[identificator in sectiune gen logo, title, subtitle, etc]`